File: src/test/java/dev/retrotv/crypt/owe/pbkdf2/Pbkdf2Test.java
package dev.retrotv.crypt.owe.pbkdf2;

import dev.retrotv.crypt.owe.OWETest;
import org.junit.jupiter.api.DisplayName;
import org.junit.jupiter.api.Test;

public class Pbkdf2Test extends OWETest {

    @Test
    @DisplayName("Pbkdf2 password encode 테스트")
    void passwordEncrypt() {
        passwordEncryptAndMatchesTest(new Pbkdf2());
    }
}

File: src/test/kotlin/dev/retrotv/crypto/owe/kdf/pbkdf2/Pbkdf2Test.kt
package dev.retrotv.crypto.owe.kdf.pbkdf2

import dev.retrotv.crypto.owe.OWETest
import dev.retrotv.crypto.owe.password.pbkdf2.Pbkdf2
import org.junit.jupiter.api.DisplayName
import org.junit.jupiter.api.Test

class Pbkdf2Test : OWETest() {
    @Test
    @DisplayName("Pbkdf2 password encode 테스트")
    fun passwordEncrypt() {
        passwordEncryptAndMatchesTest(Pbkdf2())
    }
}
